algebra/shared/src/main/scala/doodle/syntax/LoadBitmapSyntax.scala

```diff
@@ -24,16 +23,17 @@ import doodle.algebra.Picture
 import doodle.algebra.ToPicture
 
 trait LoadBitmapSyntax {
-
-  extension [S, B](specifier: S)(using loader: LoadBitmap[S, B]) {
+  extension [S](specifier: S) {
 
     /** Load a bitmap from this specifier */
-    def loadBitmap: IO[B] = loader.load(specifier)
+    def loadBitmap[B](using loader: LoadBitmap[S, B]): IO[B] =
+      loader.load(specifier)
 
     /** Load a bitmap and immediately convert to Picture. This is a convenience
       * method that combines loading and conversion.
       */
-    def loadAsPicture[Alg <: Algebra](using
+    def loadAsPicture[B, Alg <: Algebra](using
```

Comment:
I think `loadToPicture` would be a better name, as it is consistent with the `ToPicture` name.